### PR TITLE
Fix link to contributing documentation

### PR DESCRIPTION
**Pull request by teaglebuilt**
# Problem

404 when clicking on link to contributing docs.

# Action

Fix url in `CONTRIBUTING.md`

### CODE REVIEW

**Comment by teaglebuilt** [comment]
Would a pull request template be a useful pr?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ We welcome contributions to the FireFly Project in many forms, and
 there's always plenty to do!
 
 Please visit the
-[contributors guide](https://hyperledger.github.io/firefly//contributors/contributors.html) in the
+[contributors guide](https://hyperledger.github.io/firefly/contributors/contributors.html) in the
```
Comment (nguyer) [inline CONTRIBUTING.md:7]: This should actually be `https://hyperledger.github.io/firefly/contributors` or `https://hyperledger.github.io/firefly/contributors/index.html`

**Review by nguyer — CHANGES_REQUESTED** [review]
Thanks for catching this broken link, and I appreciate the PR. If the link is fixed, and DCO is resolved, I'm happy to merge. You can read here: https://github.com/hyperledger/firefly/pull/783/checks?check_run_id=6263876049 how to fix the DCO issue. Thanks again!

File: CONTRIBUTING.md (hunk 1 above)
Comment (teaglebuilt) [reply CONTRIBUTING.md:7]: Ok ill make the update

**Comment by teaglebuilt** [comment]
Found another link on the readme. Im just doing this as I am learning this architecture.

**Comment by nguyer** [comment]
Hey, I see that you tried to fix the DCO sign-off. Thanks. Unfortunately it looks like there's still one commit in this branch that does not have sign-off,  so the bot is still not happy. One suggestion is doing an interactive rebase and squashing those commits down into one commit that has sign-off, then force-pushing your branch again. That should do the trick.

**Pull request closed without merging**